lib/src/main/java/com/tubitv/media/views/TubiPlayerControlView.java

```diff
@@ -122,6 +133,12 @@ public void onAttachedToWindow() {
 //        updateAll();
     }
 
+    @Override
+    protected void onSizeChanged(int w, int h, int oldw, int oldh) {
+        super.onSizeChanged(w, h, oldw, oldh);
+        seekBarUpperBound = calculateSeekBarUpperBound();
```

Comment:
Would we want a callback here to know when this value is set?  Or is there a place I can call `getSeekBarUpperBound` after this value has been set to non-zero?